admin/tool/urlpreview/index.php

```diff
@@ -33,11 +33,16 @@
 $PAGE->set_pagelayout('standard');
 $PAGE->set_title($SITE->fullname);
 $PAGE->set_heading(get_string('menuname', 'tool_urlpreview'));
+$allowuse = has_capability('tool/urlpreview:usetool', $context);
```

Comment:
This works fine, but you can do this and the couple lines below but using require_capability instead which throw an error if you don't have it